source/modules/configuration/collectors.rst

```diff
@@ -1,6 +1,39 @@
-.. not included in any toctree, but "included" with link
+E-Mail collectors
+=================
 
-:orphan:
+E-Mail Collectors configuration can be reached from `Configuration -> Collectors` menu.
+
+A mail collector allow to import emails from a mailbox ans turn them into GLPI tickets. A :ref:`routing mechanism <collectors_rules>` will bring them to the destination entity.
+
+A mail collector is associated with an email address. It is possible to add as many collectors as you want. Of course, the more collectors you configure, the more import will take time.
+
+To declare a new collector, you will have to provide a server name, as well as connection options (IMAP or POP, SSL, TLS, certificate validation). It is possible to limit the maximum size for attachments (no import, or from 1 to 100 Mio). This configuration value is inherited from the main configuration (`Setup -> General -> Assistance`)
+A configuration option permit to switch tickets creation date between import date or email date.
+
+Optionally, you can set an archive directory in the mailbox where imported and refused emails will be stored.
+
+Collectors that have errors will be placed at the top of the list. In that case, number of successive errors will be displayed in collector configuration form.
+
+Important: collector name must be the email address used. This allows to determine the address associated to the account. This is used to prevent collector address to be added as observer in tickets.
+
+Once the form validated, you can test connection and messages retrieving using the dedicated button.
+
+An automatic action will do the import on configured collectors; time between two executions can be configured in automated task form.
+
+Another automatic action will send notifications when any collector repetitively fails to import emails. See `notifications configuration <modules/configuration/notifications>`.
```

Comment:
Are you sure this feature exists ?

Reply:
Not at all; I've just translated :D

Reply:
It seems to be the goal of the `mailgateerror` automatic action